### DISCUSSION TITLE
Have you considered making an npm package?

### DISCUSSION THREAD

**Discussion by Deja-vuuu** [simeydotme/pokemon-cards-css#17] [Q&A] [answered]
The current code structure is difficult to be used by other projects ~
Also try to test in the mobile terminal performance does not seem to be very good

**Comment by simeydotme** [comment, marked as answer]
thanks @Deja-vuuu , actually this is not really a project that is designed to be included/incorporated to others. It's not a component for download (for now). So it won't be in a package manager.

After I'm done with this project, though, I may try making a "holographic card" component which is not related to pokemon. That one would be a package.

**Reply by Deja-vuuu** [reply]
Okay, thanks for your reply. It's a great program.
